setup.cfg

```diff
@@ -72,6 +72,7 @@ install_requires =
     argcomplete>=1.10
     asgiref
     attrs>=22.1.0
+    backports.zoneinfo>=0.2.1;python_version<"3.9"
```

Comment:
I feel we don’t need to add this as an Airflow dependency. If it is installed by a dependency, the decoder can decode it, but in many situations no-one needs it.

Reply:
won't tests fail if left out?

Reply:
It can be added to `devel` in `setup.py` for tests.